academy/ibc/relayerintro.md

```diff
@@ -0,0 +1,83 @@
+
+There are two important considerations to make based on this flow. First, on the receiving chain, we need to verify the commitment proof on the source chain. This is why we need a light client to track the state of the counterparty chain (in an efficient way). Again we refer to a previous section to find out more. Second, blockchains cannot directly communicate with one another, so how do the proof and packet data arrive at the destination chain to continue the flow described above? 
+
+This is where the relayer operators come into the picture, they ensure the relaying of the packets over network infrastructure. **Relayers have access to full nodes of both source and destination chains where they can query and submit messages.** They are listening in on the chains they service for events that require an IBC packet to be sent. They run relayer software that enables them to rebuild the packet along with the proof and submit this to the destination chain. A similar process then happens upon storing the receipt on the destination chain, spurring the acknowledgement message to be sent to the source.
+
+We can see that the relayers are a crucial part of the IBC infrastructure.  Remember that relaying is permissionless and trustless (the light client verification provides the trust).
+
+## What is needed to relay?
+
+Before moving on to look at specific implmentations of relayer software, we will first introduce the general set of requirements or functionality relayer software needs to have. 
+
+1. **Information about the chains**. A relayer will relay packets between a pair of chains, so it requires some information about these chains.
+2. **Information about the path**. Once we know the chains we are relaying on, the next requirement is to know which path to relay on. Remember, the IBC protocol has three main layers of abstraction, the (light) clients, connections and channels (and ports).
+3. **A private key** to an relayar operator address on all chains that we want to relay on. Remember that a relayer needs to submit IBC messages to the chains they are relaying between (Receive, Acknowledge and Timeout), which mostly requires a fee. Therefore the relayer operator address needs to have funds.
+4. **Ability to query and submit messages (or transactions)**. As mentioned already, chains do not communicate directly between one another. It is the relayer's job to listen for events related to a packet commitment. They can do this by subscribing to these events via the Tendermint websocket and query the proofs via the Tendermint RPC endpoint. We also have the ability to query and create through transactions: clients, connections, channels. We have transactions to update and upgrade light clients, submit misbehaviour. And of course, we have the transactions that allow to relay packets and acknowledgements or timeouts. For a more detailed look, we will look at the command lists for the Go and Hermes relayers.
+<HighlightBox type="tip">
```

Comment:
This higlightbox is not showing up correctly yet

Reply:
Should be fixed with CitMC's last commit.